### PR TITLE
better enum support + fix for enum34 IntEnum quirk

### PR DESCRIPTION
**Pull request by ezarowny**
See https://bitbucket.org/stoneleaf/enum34/issues/17/difference-between-enum34-and-enum-json for enum34 quirk

### CODE REVIEW

File: rollbar/lib/transforms/__init__.py [hunk 1]
```diff
@@ -79,12 +79,19 @@ def string_handler(s, key=None):
...
-            return do_transform('number', o, key=key)
-
         if isinstance(o, bool):
             return do_transform('boolean', o, key=key)
 
+        # There is a quirk in the current version (1.1.6) of the enum
+        # backport enum34 which causes it to not have the same
+        # behavior as Python 3.4+. One way to identify IntEnums is that
+        # they are instances of numbers but not number types.
+        if isinstance(o, integer_types + (float,)):
```
Comment (coryvirok) [inline rollbar/lib/transforms/__init__.py]: It's probably worth creating a global `number_types` so we aren't constantly creating new tuples.

**Comment by coryvirok** [comment]
👍

File: rollbar/lib/transforms/__init__.py (hunk 1 above)
Comment (ezarowny) [inline rollbar/lib/transforms/__init__.py]: yeah that's probably true - not a bad idea at all

**Comment by ezarowny** [comment]
cc @dlsteuer

**Pull request merged**